practice-app/website/api/event.py

```diff
@@ -76,5 +79,74 @@ def event():
         except exc.SQLAlchemyError as e:
             db.session.rollback()
             return "Try Later", 403
-        
+
         return jsonify(new_event.serialize()), 201
+
+
+def check_comment_has_text(message):
+    if len(message) < 1:
+        return False
+    else:
+        return True
+
+"""
+This method handles all requests for adding a discussion post and getting all
+discussion posts in order to show those in the event's discussion page.
+"""
+@events.route('<event_id>/discussions', methods=['GET', 'POST'])
+@swag_from('doc/discussions_GET.yml', methods=['GET'])
+@swag_from('doc/discussions_POST.yml', methods=['POST'])
+def discussionForEvent(event_id):
+
+    if request.method == 'POST':
+
+        if event_id < 0:
+            return "Wrong parameters", 401
```

Comment:
This should be 400, 401 means "unauthorized".

Reply:
Corrected, thank you.